### DISCUSSION TITLE
I have to keep reentering my key.

### DISCUSSION THREAD

**Discussion by jjh63084** [aclap-dev/video-downloadhelper#1575] [Help and Q&A]
### Your question

I have to keep reentering my key. The box goes blank every time I try downloading a video. This seems to happen after updates.

### Addon & CoApp version and details

I have to keep reentering my key. The box goes blank every time I try downloading a video. This seems to happen after updates.

**Comment by jcfrog** [comment]
It's probably some settings options in your browser, or cleaners that clear local storage.

**Reply by jcfrog** [reply]
You can have a look here: https://github.com/aclap-dev/video-downloadhelper/wiki/Why-am-I-asked-to-pay-again%3F
